section_to_course/management/commands/section_to_course.py

```diff
@@ -0,0 +1,54 @@
+"""
+Django command for converting a section into a course.
+"""
+
+from django.contrib.auth import get_user_model
+from django.core.management.base import BaseCommand
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from opaque_keys.edx.locator import BlockUsageLocator
+from xmodule.modulestore.exceptions import ItemNotFoundError
+
+from section_to_course.utils import paste_from_template
+
+User = get_user_model()
+
+
+class Command(BaseCommand):
+    """
+    Management command to convert a section into a course.
+    """
+
+    help = 'Converts a section into a course'
+
+    def add_arguments(self, parser):
+        parser.add_argument('destination_course_id', type=str)
+        parser.add_argument('source_section_id', type=str)
```

Comment:
Let's use the `source` before the `destination` in the management command. It is more intuitive (same as [here](https://github.com/open-craft/section-to-course/pull/1#discussion_r1168494486)).